bundles/org.openhab.binding.automower/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,167 @@
+				<label>Automower Id</label>
+				<description>The Id of an automower as used by the Automower Connect Api to identify a mower</description>
+			</parameter>
+			<parameter name="pollingInterval" type="integer" required="false" unit="s">
+				<label>Polling Interval</label>
+				<default>600</default>
+				<description>How often the current automower state should be polled in seconds</description>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="nameType">
+		<item-type>String</item-type>
+		<label>Automower Name</label>
+		<description>The channel providing the automower name</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="modeType">
+		<item-type>String</item-type>
+		<label>Mode</label>
+		<description>The current mode</description>
+		<state readOnly="true">
+			<options>
+				<option value="MAIN_AREA">Main Area</option>
+				<option value="SECONDARY_AREA">Secondary Area</option>
+				<option value="HOME">Home</option>
+				<option value="DEMO">Demo</option>
+				<option value="UNKNOWN">Unknown</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="activityType">
+		<item-type>String</item-type>
+		<label>Activity</label>
+		<description>The current activity</description>
+		<state readOnly="true">
+			<options>
+				<option value="UNKNOWN">Unknown</option>
+				<option value="NOT_APPLICABLE">N/A</option>
+				<option value="MOWING">Mowing</option>
+				<option value="GOING_HOME">Returning to charging station</option>
+				<option value="CHARGING">Charging</option>
+				<option value="LEAVING">Leaving charging station</option>
+				<option value="PARKED_IN_CS">Parked in charging station</option>
+				<option value="STOPPED_IN_GARDEN">Stopped in garden</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="stateType">
+		<item-type>String</item-type>
+		<label>State</label>
+		<description>The current state</description>
+		<state readOnly="true">
+			<options>
+				<option value="UNKNOWN">Unknown</option>
+				<option value="NOT_APPLICABLE">N/A</option>
+				<option value="PAUSED">Paused by user</option>
+				<option value="IN_OPERATION">Working</option>
+				<option value="WAIT_UPDATING">Downloading new firmware</option>
+				<option value="WAIT_POWER_UP">Booting mower</option>
+				<option value="RESTRICTED">Waiting</option>
+				<option value="OFF">Off</option>
+				<option value="STOPPED">Stopped- Manual intervention required</option>
+				<option value="ERROR">Error</option>
+				<option value="FATAL_ERROR">Fatal error</option>
+				<option value="ERROR_AT_POWER_UP">Boot error</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="lastUpdateType">
+		<item-type>DateTime</item-type>
+		<label>Last Update</label>
+		<description></description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="batteryType">
+		<item-type>Number</item-type>
```

Comment:
If this number is supposed to be a percentage then you should change this a `Number:Dimensionless` and then use QuantityType<Dimensionless> and percentage units.

Reply:
Sure